### PR TITLE
fix(VTooltip): bring back --fixed in VTooltip.sass

### LINKED ISSUE #8894: [Bug Report] Page scroll down causes incorrect tooltip position inside a dialog
### Environment
**Vuetify Version:** 2.0.15
**Vue Version:** 2.6.10
**Browsers:** Chrome 76.0.3809.132
**OS:** Mac OS 10.14.5

### Steps to reproduce
Step 1. Put a button with tooltip inside a dialog;

Step 2. Put the dialog activator button outside the viewport (for example, to make the activator button only visible after page scroll down);

Step 3. Scroll down to see the dialog activator button, click it to open the dialog;

Step 4. Hover to the tooltip's button inside that dialog, see the incorrect tooltip position;
 

### Expected Behavior
The tooltip should appear around the button

### Actual Behavior
The tooltip is far away from the button, in case of page scroll down, it will appear somewhere above the button, if the scroll down distance is long enough, the tooltip may be totally outside the viewport

### Reproduction Link
<a href="https://codepen.io/1isten/pen/RwbQgKW" target="_blank">https://codepen.io/1isten/pen/RwbQgKW</a>

### PR DESCRIPTION
**Pull request by fly1ted**
--fixed somehow got lost, bringing it back

See [VTooltip.ts#L189](https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VTooltip/VTooltip.ts#L189)

Related #6460
fixes #8894

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [ ] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [ ] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VTooltip/VTooltip.sass [hunk 1]
```diff
@@ -17,6 +17,9 @@
     width: auto
     opacity: 1
     pointer-events: none
+    
+    &[class*="--fixed"]
```
Comment (sh7dm) [inline packages/vuetify/src/components/VTooltip/VTooltip.sass]:
```suggestion
    &[class*="-fixed"]
```

Comment (jacekkarczmarczyk) [reply packages/vuetify/src/components/VTooltip/VTooltip.sass]: why?

Comment (fly1ted) [reply packages/vuetify/src/components/VTooltip/VTooltip.sass]: both are ok, but [class*="-fixed"] is consistent with the siblings

Comment (jacekkarczmarczyk) [reply packages/vuetify/src/components/VTooltip/VTooltip.sass]: And `-` version is not consistent with other components, for example https://github.com/vuetifyjs/vuetify/blob/fae3f271f484e84169b0563a07682f76182b0e2f/packages/vuetify/src/components/VMenu/VMenu.sass#L28

Comment (fly1ted) [reply packages/vuetify/src/components/VTooltip/VTooltip.sass]: agreed

**Pull request merged**